assets/scripts/segments/view.js

```diff
@@ -154,6 +154,8 @@ export function getVariantInfoDimensions (variantInfo, actualWidth = 0) {
     }
   }
 
+  // what does graphics left, right mean?
```

Comment:
@louh ok so turns out i had left more notes as comments on the issue and less work on this PR, so feel free to close this if you want